inference-engine/src/mkldnn_plugin/nodes/mkldnn_shuffle_channels_node.cpp

```diff
@@ -0,0 +1,712 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
[random place in the file] Please consider adding CPU specific single layer tests for ShuffleChannel operation which will cover different ISA and layouts. See https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/single_layer_tests/interpolate.cpp as an example. 
Existing shared ShCh test for reference: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/shuffle_channels.cpp

Reply:
Fixed. Thanks Dmitry!